### PR TITLE
Fix update CliVar submission ID manually

### LINKED ISSUE #3725: Not possible to update the ClinVar submission ID manually
This doesn't work any more:

image

I must have introduced a bug in one of my latest PRs! 😢

### PR DESCRIPTION
**Pull request by northwestwitch**
Fix #3725 

I've separated the form elements into 2 distinct forms

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. Install on stage or locally and make sure that it assigning a submission ID works, both manually and via the API

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by DN, CR

### CODE REVIEW

File: scout/server/blueprints/clinvar/templates/clinvar_submissions.html [hunk 1]
```diff
@@ -103,30 +104,27 @@ <h5>Upload the files to ClinVar</h5>
...
-            <div class="col-3">
-              <a href="{{ url_for('clinvar.clinvar_validate', submission=subm_obj._id, save_id='y') }}" class="btn btn-sm btn-primary text-white">Validate and obtain ID from ClinVar</a>
-            </div>
-            {% if subm_obj.clinvar_subm_id and subm_obj.status != "submitted" %}
-              <div class="col-2">
-                <button type="button" class="btn btn-sm btn-success form-control" name="update_submission" value="api_submit" data-bs-toggle="modal" data-bs-target="#submitModal">Submit to ClinVar</button>
-              </div>
+      </div>
+      <div class="row">
+        <table>
```
[the remark this answers is not in the document]
Comment (dnil) [reply scout/server/blueprints/clinvar/templates/clinvar_submissions.html:109]: Same here, not a big fan of introducing tables just for layout, but on the other hand Im also not a big fan of the decision to make HTML forms play into the design. Would be beneficial if they could allow for a free separation of interaction and layout. 🙄

**Review by dnil — APPROVED** [review]
Works again! 💯 

Tables for layout are a bit dated, and have issues for accessibility - which afaik is not an issue for us right now, but could be at one point. At least provide aria roles to mark it as layout?

File: scout/server/blueprints/clinvar/templates/clinvar_submissions.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (northwestwitch) [reply scout/server/blueprints/clinvar/templates/clinvar_submissions.html:109]: > Im also not a big fan of the decision to make HTML forms play into the design.

Yes, it complicates things a lot!

**Comment by northwestwitch** [comment]
> Works again! 💯
> 
> Tables for layout are a bit dated, and have issues for accessibility - which afaik is not an issue for us right now, but could be at one point. At least provide aria roles to mark it as layout?

Done, and SonarCloud complains, same as in the other PR. I'm going to keep it like this and ignore SonarCloud for now. We can fix it later.

**Pull request merged**